configure

```diff
@@ -1607,12 +1594,14 @@ case "${ARCH}" in
             check_sse42_intrinsics
 
             if test ${HAVE_SSE42CRC_INTRIN} -eq 1 || test ${HAVE_SSE42CRC_INLINE_ASM} -eq 1; then
-                CFLAGS="${CFLAGS} -DX86_SSE42"
-                SFLAGS="${SFLAGS} -DX86_SSE42"
-
                 if test ${HAVE_SSE42CRC_INTRIN} -eq 1; then
-                  CFLAGS="${CFLAGS} -DX86_SSE42_CRC_INTRIN"
-                  SFLAGS="${SFLAGS} -DX86_SSE42_CRC_INTRIN"
+                    CFLAGS="${CFLAGS} -DX86_SSE42"
```

Comment:
Indent seems off here.

Reply:
True, but the wrong indent was actually in the if below.
The old indent here was 2 spaces instead of 4, so this one is correct.

Reply:
Yeah I didn't look to see which is the wrong one.